pkg/e2e/autoscaler/autoscaler.go

```diff
@@ -385,7 +394,10 @@ var _ = g.Describe("[Feature:Machines][Serial] Autoscaler should", func() {
 		}
 		scaleUpCounter := newScaleUpCounter(eventWatcher, 0, scaledGroups)
 		maxNodesTotalReachedCounter := newMaxNodesTotalReachedCounter(eventWatcher, 0)
-		workload := newWorkLoad()
+		bytes, ok := memCapacity.AsInt64()
+		o.Expect(ok).Should(o.BeTrue())
+		// set job load to 70% of available memory per instance.
+		workload := newWorkLoad(int32(maxNodesTotal+1), resource.MustParse(fmt.Sprintf("%v", bytes*70/100)))
```

Comment:
Not: I think it's safe to directly use `0.7` coefficient instead of explicitly describing 70%.

Otherwise, 70% is sufficient. Enough capacity and not too much to cover existing and "to-be-created" use cases.